src/language-service/src/schemas/types.ts

```diff
@@ -165,7 +165,7 @@ export type PositiveInteger = number;
 export type Port = number;
 
 /**
- * @TJS-pattern ^!secret\s([a-zA-Z0-9_-]+)$
+ * @TJS-pattern ^!secret|([a-zA-Z0-9_-]+)$
```

Comment:
That is replacing the space after secret with a or?

The format of a secret is `!secret key`, how would this change match that?

Reply:
Ah yeah that is true. My thinking was needs a !secret <key regrex> OR !secret <string>

Reply:
Yes, but be sure to test it.